docs/dev/annual_updates.rst

```diff
@@ -156,57 +142,140 @@ B. FERC Form 1
 transform step on all tables of the new year of data and debug any failures. If any new
 tables were added in the new year of data you will need to add a new transform function
 for the corresponding dataframe. If new columns have been added, they should also be
 inspected for cleanup.
 
+.. note::
+
+    The next time we update EIA we should probably do so in the new transform framework.
+
 B. FERC Form 1
 ^^^^^^^^^^^^^^
 Some FERC 1 tables store different variables in different rows instead of or in addition
-to using columns. Rows are identified by ``row_number``. What row number corresponds to
-which variable changes from year to year.  We catalog this correspondence in the FERC 1
-row maps, a collection of CSV files stored under
-``src/pudl/package_data/ferc1/row_maps`` and organized by original FERC 1 DB table name.
-
-**4.B.1)** Check whether the data associated with a given row number has changed
-by looking at the table's entries in the ``f1_row_lit_tbl`` table. This table stores the
-descriptive strings associated with each row in the FERC Form 1, and also indicates the
-last year that the string was changed in the ``row_chg_yr`` column. The
-``devtools/ferc1/ferc1-new-year.ipynb`` notebook can make this process less tedious.
-
-**4.B.2)** The ``plant_kind`` and ``construction_type`` fields in the
-``plants_steam_ferc1`` table and the ``fuel_type`` and ``fuel_unit`` fields in the
-``fuel_ferc1`` table are reported as freeform strings and need to be converted to simple
-categorical values to be useful. If the new year of data contains strings that have
-never been encountered before, they need to be added to the string cleaning dictionaries
-defined in :mod:`pudl.transform.ferc1`. The ``devtools/ferc1/ferc1-new-year.ipynb``
-notebook and :func:`pudl.helpers.find_new_ferc1_strings` will help with this process.
-Every string observed in these fileds should ultimately be mapped to one of the defined
-categories.
-
-5. Update the PUDL DB schema
-----------------------------
-**5.1)** If new columns or tables have been added, you will need to update the
-PUDL DB schema, defining column types, giving them meaningful descriptions, applying
-appropriate ENUM constraints, etc. This happens in the :mod:`pudl.metadata` subpackage.
-Otherwise when the system tries to write dataframes into SQLite, it will fail.
+to using columns. In the Pre-2021 data (from the DBF files), rows are identified by
+``row_number``, and the row number that corresponds to a given variable changes from
+year to year. We cataloged this correspondence in the FERC 1 row maps, a collection of
+CSV files stored under ``src/pudl/package_data/ferc1/row_maps`` and organized it by
+original FERC 1 DB table name.
+
+The FERC 1 data from 2021 onwards (XBRL files) is organized into a taxonomy that keeps
+it relatively consistent year to year. We keep track of the connection between DBF and
+XBRL tables in the ``src/pudl/package_data/ferc1/dbf_to_xbrl_tables.csv`` and
```

Comment:
I was actually wondering about this. Is the `dbf_to_xbrl_tables.csv` even necessary? Seems like it's just misleading...

Reply:
But I added some clarifying language in here.

Reply:
`dbf_to_xbrl_tables.csv` isn't in use at all right now. I could imagine using it instead of `TABLE_NAME_MAP_FERC1`, but right now it feels dangly and misleading imo. although it does contain all of the tables so its a helpful reference so idk if deleting it altogether is a good idea.

Reply:
Ok we can leave it for now.